scripts/configs/interactive_learning.yaml

```diff
@@ -52,14 +52,11 @@ ENVS:
 ARGS: {}
 FLAGS:  # common args
   excluded_predicates: "Covers,Holding"
-  interactive_score_threshold: 0.05
-  num_online_learning_cycles: 1000  # way too many cycles
-  online_learning_max_transitions: 10000  # want this to be stop signal
-  interactive_num_requests_per_cycle: 10
+  num_online_learning_cycles: 100  # way too many cycles
```

Comment:
did you intentionally reduce both of these?

Reply:
yeah, those were the original settings

Reply:
Good catch!